### PR TITLE
Casket additions and adjustments

### PR DESCRIPTION
**Pull request by Omnione**
• Moved loot tables to casket_loot.lua as it was a bloating caskets.lua.
• Added all remaining zone mixins, treasure_casket.lua's and checked zone ID text tables.
• Added all remaining loot tables for zones added.
• Fixed an issue where a hint would return an incorrect number.
• Fixed an issue where you cannot enter the number 10 as an input.

**_I affirm:_**
- [X] that I agree to Project Topaz's [Limited Contributor License Agreement](http://project-topaz.com/blob/release/CONTRIBUTOR_AGREEMENT.md), as written on this date
- [X] that I've _tested my code_ since the last commit in the PR, and will test after any later commits

### CODE REVIEW

File: scripts/globals/caskets.lua [hunk 1]
```diff
@@ -243,8 +98,10 @@ local function timeElapsedCheck(npc)
     local spawnTime   = os.time() + 360000 -- defualt time in case no var set.
     local timeTable   = {0, 0, 0}          -- HOURS,MINUTES,SECONDS.
 
-    spawnTime = npc:getLocalVar("[caskets]SPAWNTIME")
-
+    if npc ~= nil then
+        spawnTime = npc:getLocalVar("[caskets]SPAWNTIME")
+    end
+
```
Comment (ibm2431) [inline scripts/globals/caskets.lua]: Thoughts on doing:
```lua
if npc == nil then
    return false
end
```
To immediately abort the rest of the function?

File: scripts/globals/casket_loot.lua [hunk 2]
```diff
@@ -0,0 +1,4462 @@
...
+            { 690, 12706}, -- Iron Mittens
+            {1100, 12731}, -- Velvet Cuffs
+            { 860, 12827}, -- Cuir Trousers
+            { 720, 12836}, -- Iron Subligar
+            { 490, 12859}, -- Velvet Slops
+            {1120, 12955}, -- Cuir Highboots
+            { 390, 12987}, -- Ebony Sabots
+        }
+    },
+        [tpz.zone.QUICKSAND_CAVES] =
```
Comment (ibm2431) [inline scripts/globals/casket_loot.lua]: Got an extra indent on this key

Comment (Omnione) [reply scripts/globals/casket_loot.lua]: stupid copy pasta

File: scripts/globals/caskets.lua (hunk 1 above)
Comment (Omnione) [reply scripts/globals/caskets.lua]: thats a good call i'll do that

Comment (Omnione) [reply scripts/globals/caskets.lua]: im gonna add a check the see if the casket has a local var set for its spawn time too.

File: scripts/globals/casket_loot.lua (hunk 2 above)
Comment (Omnione) [reply scripts/globals/casket_loot.lua]: Fixed

File: scripts/globals/caskets.lua (hunk 1 above)
Comment (Omnione) [reply scripts/globals/caskets.lua]: Added the the suggested change plus a check for the local var.

**Review by ibm2431 — APPROVED** [review]
Thanks, Omni!

**Pull request merged**